apps/music/js/ui/views/list_view.js

```diff
@@ -60,8 +60,8 @@ var ListView = {
     this.lastFirstLetter = null;
     this.anchor.innerHTML = '';
     this.anchor.style.height = 0;
-    this.view.scrollTop = 0;
-    this.hideSearch();
+    this.searchInput.value = '';
+    this.view.scrollTop = this.anchor.offsetTop;
```

Comment:
I'm going to assume that the changes in this file are part of some other bug, and that you don't need me to review them.  I don't know what's going on here.